storage/src/main/scala/coop/rchain/storage/regex/PathRegex.scala

```diff
@@ -95,15 +219,14 @@ object PathRegex {
     } else {
       //we need to transform our string to UTF-8 (internally, Java/Scala uses UTF-16)
       str
-        .map(c => {
+        .map(c =>
           if (uriAllowedChars.contains(c)) {
             c.toString
           } else {
             c.toString
               .getBytes(StandardCharsets.UTF_8)
-              .map(b => "%02X".format(b))
+              .map(b => "%%%02X".format(b))
```

Comment:
Curious: why the extra `%%`?

Reply:
That's a mix of uri standard and Scala (c-style) format string. To include hex-char it should be represented in format string as %## where xx - char hex code. And to add % symbol before char code - I need to escape it with one extra %. As a result first %%->% and %02X->##

Reply:
Gotcha